10_functions/Uppercase.js

```diff
@@ -0,0 +1,6 @@
+var phrase = "i love javascripting, it is so much fun";
+
+function uppercase(){
+    return phrase.toUpperCase();
```

Comment:
`phrase` is a variable outside of the function. Instead of using `phrase ` here, you should add a parameter to the function, and use the parameter name instead of `phrase`.